frontend/src/types/QuestionMetadataTypes.ts

```diff
@@ -6,6 +6,13 @@ export interface TextMetadata {
   text: string;
 }
 
+export interface ImageMetadata {
+  previewUrl: string;
+  file: File | undefined;
+}
+
+export type ImageUploadMetadata = Omit<ImageMetadata, "previewUrl">;
```

Comment:
i'm not sure if it's better to keep as is (having two types) or pass the previewUrl to the backend for the sake of not having two types in FE. but the field will never be used. for now, considering the only change is with formatQuestionsRequest i feel like its fine. but lmk if you think of any potential issues / a cleaner way